lib/Rest/VstRequest.cpp

```diff
@@ -66,6 +66,15 @@ VstRequest::VstRequest(ConnectionInfo const& connectionInfo,
   parseHeaderInformation();
 }
 
+arangodb::velocypack::StringRef VstRequest::rawPayload() const {
+  if (_buffer.size() < _payloadOffset) {
```

Comment:
Changing this to
```suggestion
  if (_buffer.size() <= _payloadOffset) {
```
will be even better, though the previous approach will already prevent disaster.